src/App.stories.tsx

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
+/** @jsxRuntime classic */
```

Comment:
I think this is only required from [Babel 7.9.0](https://babeljs.io/docs/en/babel-preset-react/). Do you plan on updating that?